packages/forma-36-website/yarn.lock

```diff
@@ -778,6 +785,31 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
+"@contentful/forma-36-fcss@^0.0.34":
```

Comment:
What happens if you run `yarn` from the root of the repo? Wondering if these disappear from the lock file again (I removed them in https://github.com/contentful/forma-36/pull/447)

Reply:
I will try it again!

Reply:
Well, I rebased and got the yarn.lock from master. Then I ran `yarn` from the root but the .lock it's still here. What am I missing here?